client/src/components/newColumn/Column.js

```diff
@@ -0,0 +1,78 @@
+import React, { useContext } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+
+import Card from "../newCard/Card";
+//check how to use Cards in column data
+// import { fakeData } from "../../dragAndDrop/fakeData";
+
+//materialUi
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+
+//Context APi
+import { UserContext } from "../../userContext/userContext";
+
+//
+import { Droppable } from "react-beautiful-dnd";
+
+const Column = props => {
+  const classes = useStyles(props);
+  const { columnName, cards, createNew, columnId } = props;
+  const { value1 } = useContext(UserContext);
+  const [taskState, setTaskState] = value1;
+
+  const render = (
+    <div className={classes.root}>
+      <TextField>enter Text</TextField>
+      <Button className={classes.btn} variant='contained' color='secondary'>
+        Add new
+      </Button>
+    </div>
+  );
+
+  if (createNew) {
```

Comment:
could use the ternary operator here

Reply:
Thanks for your comment!
I updated the entire code, but still this part reminds.
What part do you mention?